src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -137,6 +137,89 @@ hello
         End Sub
 
         <Fact>
+        Public Sub TupleTypeBindingTypeChar()
```

Comment:
>TupleTypeBindingTypeChar [](start = 19, length = 24)

I think we should verify that we have no unexpected diagnostics for this scenario.